packer_templates/ubuntu/ubuntu-22.04-arm64.json

```diff
@@ -0,0 +1,93 @@
+            "disk_size": "{{user `disk_size`}}",
+            "guest_os_type": "ubuntu",
+            "http_directory": "{{user `http_directory`}}",
+            "iso_checksum": "{{user `iso_checksum`}}",
+            "iso_url": "{{user `mirror`}}/{{user `mirror_directory`}}/{{user `iso_name`}}",
+            "memory": "{{ user `memory` }}",
+            "output_directory": "{{ user `build_directory` }}/packer-{{user `template`}}-parallels",
+            "parallels_tools_flavor": "lin-arm",
+            "prlctl_version_file": ".prlctl_version",
+            "shutdown_command": "echo 'vagrant' | sudo -S shutdown -P now",
+            "ssh_password": "vagrant",
+            "ssh_port": 22,
+            "ssh_timeout": "10000s",
+            "ssh_username": "vagrant",
+            "type": "parallels-iso",
+            "vm_name": "{{ user `template` }}"
+        }
+    ],
+    "post-processors": [
+        {
+            "output": "{{ user `build_directory` }}/{{user `box_basename`}}.{{.Provider}}.box",
+            "type": "vagrant"
+        }
+    ],
+    "provisioners": [
+        {
+            "environment_vars": [
+                "HOME_DIR=/home/vagrant",
+                "http_proxy={{user `http_proxy`}}",
+                "https_proxy={{user `https_proxy`}}",
+                "no_proxy={{user `no_proxy`}}"
+            ],
+            "execute_command": "echo 'vagrant' | {{.Vars}} sudo -S -E sh -eux '{{.Path}}'",
+            "expect_disconnect": true,
+            "scripts": [
+                "{{template_dir}}/scripts/update.sh",
+                "{{template_dir}}/../_common/motd.sh",
+                "{{template_dir}}/../_common/sshd.sh",
+                "{{template_dir}}/scripts/networking.sh",
+                "{{template_dir}}/scripts/sudoers.sh",
+                "{{template_dir}}/scripts/vagrant.sh",
+                "{{template_dir}}/../_common/virtualbox.sh",
+                "{{template_dir}}/scripts/vmware.sh",
+                "{{template_dir}}/../_common/parallels.sh",
+                "{{template_dir}}/scripts/hyperv.sh",
+                "{{template_dir}}/scripts/cleanup.sh",
+                "{{template_dir}}/../_common/minimize.sh"
+            ],
+            "type": "shell"
+        }
+    ],
+    "variables": {
+        "box_basename": "ubuntu-22.04-arm64",
+        "build_directory": "../../builds",
+        "build_timestamp": "{{isotime \"20060102150405\"}}",
+        "cpus": "2",
+        "disk_size": "65536",
+        "git_revision": "__unknown_git_revision__",
+        "guest_additions_url": "",
+        "headless": "",
+        "http_directory": "{{template_dir}}/http",
+        "http_proxy": "{{env `http_proxy`}}",
+        "https_proxy": "{{env `https_proxy`}}",
+        "hyperv_generation": "2",
+        "hyperv_switch": "bento",
+        "iso_checksum": "c209ab013280d3cd26a344def60b7b19fbb427de904ea285057d94ca6ac82dd5",
+        "iso_name": "ubuntu-22.04-live-server-arm64.iso",
```

Comment:
This should be updated to `"iso_name": "ubuntu-22.04.1-live-server-arm64.iso",`.

```
┌─(~/Code/bento/packer_templates/ubuntu)(ubuntu-22.04-arm64 ??:2 M:1 S:14)
└─(130)─ packer build --only=parallels-iso ubuntu-22.04-arm64.json
parallels-iso: output will be in this color.

==> parallels-iso: Retrieving ISO
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04-live-server-arm64.iso
==> parallels-iso: Trying http://cdimage.ubuntu.com/releases/22.04/release/ubuntu-22.04-live-server-arm64.iso?checksum=sha256%3Ac209ab013280d3cd26a344def60b7b19fbb427de904ea285057d94ca6ac82dd5
==> parallels-iso: Download failed bad response code: 404
==> parallels-iso: error downloading ISO: [bad response code: 404]
Build 'parallels-iso' errored after 5 seconds 905 milliseconds: error downloading ISO: [bad response code: 404]

==> Wait completed after 5 seconds 906 milliseconds

==> Some builds didn't complete successfully and had errors:
--> parallels-iso: error downloading ISO: [bad response code: 404]

==> Builds finished but no artifacts were created.
```

Reply:
Thanks! I've pushed an update to use the new iso version/checksum.

Reply:
Thanks so much for this PR!